src/GraphQL.Tests/Execution/InputConversionSystemTextJsonTests.cs

```diff
@@ -1,5 +1,6 @@
 namespace GraphQL.Tests.Execution;
 
+[Collection("StaticTests")]
```

Comment:
The attribute was defined on the base class but I'm guessing it must not propagate to derived classes.